components/public-api/gitpod/v1/organization.proto

```diff
@@ -159,26 +161,32 @@ message UpdateOrganizationSettingsRequest {
   // true.
   repeated string restricted_editor_names = 6;
 
-  // Specifies whether restricted_workspace_classes should be updated.
+  // Specifies whether restricted_workspace_classes should be updated
   optional bool update_restricted_editor_names = 7;
 
   // pinned_editor_versions updates the pinned version for the corresponding
-  // editor.
+  // editor
   map<string, string> pinned_editor_versions = 8;
 
-  // Specifies whether pinned_editor_versions should be updated.
+  // Specifies whether pinned_editor_versions should be updated
   optional bool update_pinned_editor_versions = 9;
 
-  // default_role is the default role for new members in the organization.
+  // default_role is the default role for new members in the organization
   optional string default_role = 10;
 
   // timeout_settings are the settings for workspace timeouts
   optional TimeoutSettings timeout_settings = 11;
 
   repeated RoleRestrictionEntry role_restrictions = 12;
 
-  // Specifies whether role_restrictions should be updated.
+  // update_role_restrictions specifies whether role_restrictions should be updated
   optional bool update_role_restrictions = 13;
+
+  // update_max_parallel_running_workspaces specifies whether max_parallel_running_workspaces should be updated
+  optional bool update_max_parallel_running_workspaces = 14;
```

Comment:
I'm really not fond of this additional bool, just to communicate "apply: true|false", but as it seems to be an ongoing pattern here, fine with me. :+1: